app/views/provider_interface/start_page/show.html.erb

```diff
@@ -24,10 +24,13 @@
   </div>
 </div>
 
-<%= render ProviderInterface::InformationBannerComponent.new %>
-
 <div class="govuk-width-container">
   <section class="app-section">
+    <div class="govuk-grid-row">
```

Comment:
do we need those wrappers as they exist in the component, the original looked like this: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4635/files#diff-b8517c38a7d309ffcbdf847eb27a7451e081f4b1270d800954e334b4c89b640dL28

Reply:
nvm removed in next commit 👍